syncdiff

```diff
@@ -61,6 +61,20 @@ sub display_help {
 	print "If you are running this you had better be John 'Warthog9' Hawley\n";
 }
 
+sub debug_mode {
+	require File::Find::Rule;
```

Comment:
Noting that you'll also have to modify the yacc and regenerate the lexer and parser to add a config file option.  That should likely be done as a separate commit.